lib/Dialect/Polynomial/Conversions/PolynomialToModArith/PolynomialToModArith.td

```diff
@@ -0,0 +1,25 @@
+#ifndef LIB_DIALECT_POLYNOMIAL_CONVERSIONS_POLYNOMIALTOMODARITH_POLYNOMIALTOMODARITH_TD_
+#define LIB_DIALECT_POLYNOMIAL_CONVERSIONS_POLYNOMIALTOMODARITH_POLYNOMIALTOMODARITH_TD_
+
+include "mlir/Pass/PassBase.td"
+include "lib/Dialect/ModArith/IR/ModArithDialect.td"
+
+def PolynomialToModArith : Pass<"polynomial-to-mod-arith", "ModuleOp"> {
+  let summary = "Lower `polynomial` to `mod-arith`.";
+
+  let description = [{
+    This pass lowers the `polynomial` dialect to `mod-arith`, with
+    affine and tensor.
```

Comment:
I think this pass deserves a bit more detailed documentation.

Reply:
Details added.